source/js/components/news/news.jsx

```diff
@@ -3,6 +3,13 @@ import PropTypes from 'prop-types';
 import moment from 'moment';
 import env from '../../../../env.json';
 
+let networkApiDomain = env.NETWORK_API_DOMAIN
+
+// HEROKU_APP_DOMAIN is used by review apps
+if (!networkApiDomain && env.HEROKU_APP_NAME) {
```

Comment:
If Heroku is cloning the environmental variables from staging, can review apps unset a single one automatically?

Reply:
I've actually elected in the app.json to not inherit from staging. all that needs to be set should be set automatically. In the case of `HEROKU_APP_NAME` we have no control over what variable this value is assigned to and must check for it manually.